test/test_artifacts/v1/jupyter-ydoc.test.Dockerfile

```diff
@@ -0,0 +1,6 @@
+ARG COSMOS_IMAGE
+FROM $COSMOS_IMAGE
+
+ARG MAMBA_DOCKERFILE_ACTIVATE=1
+
+CMD ["python", "-c", "import jupyter_ydoc"]
```

Comment:
Same here

Reply:
For jupyter_ydoc, we should add tests like https://github.com/jupyter-server/jupyter_ydoc/blob/main/tests/test_ydocs.py#L8-L11 to Line 6, so that we can validate its functionalities.